demos/jans-tent/cert.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
?

Reply:
this is a self-signed cert used for testing in flask server here:
https://github.com/JanssenProject/jans/blob/f52d5613ea14a02bcdff59c77e81accfa607c99b/demos/jans-tent/main.py#L27

Reply:
removed and added instructions to doc